packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -1164,11 +1166,9 @@ export class ProjectService {
                             metricQuery: JSON.stringify(metricQuery),
                             type: warehouseClient.credentials.type,
                         },
-                        async () => {
-                            await sshTunnel.testConnection();
```

Comment:
what's the reason for removing the ssh test? Is it needed to open the connection to the tunnel ?

Reply:
I added rhis test last week to mitigate the issue . It is not needed anymore.

Afaik, if reconnect is true, connection is open (reconnecting) but tunnel might be closed. If reconmect is false, connection and tunnels will be closed